src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ExceptionHandlerTests.kt

```diff
@@ -158,7 +156,7 @@ class ExceptionHandlerTests : SecureIntegrationTests()
     fun `does not include original message from handle exceptions`()
     {
         val sut = HintExceptionHandler(RandomErrorCodeGenerator(), ConfiguredAppProperties())
-        val result = sut.handleException(PSQLException("some message", mock()), mock())
+        val result = sut.handleException(PSQLException("some message", null), mock())
```

Comment:
Why was this change needed?

Reply:
Yes, I updated Postgres. The newer version does not support mocking for PSQLState.